recipes/dalex/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "dalex" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1a2f70259c4df7d1ae7c075d8b4a66a21586c0f9764a725cf367bc0d3a4b46f3
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
Do you need a compiler? You either need to remove the compiler or remove `noarch: python` (as it needs to be platform dependent if you're compiling code).

If you don't need a compiler you should probably also:
* remove numpy from `host`
* Remove pin_compatible on numpy from `run` in favour of `>=1.18.4`
* Remove the >=3.6 on the Python dependencies in hosr and run

If you do need a compiler you should either:
* Remove the numpy version constraint in host so the global pin is used instead (1.16 currently)
* Or use `- numpy =1.18` for now (only if absolutely necessary, it'll cause a maintance burden to try and keep a compatible version).

Reply:
Thanks, the overall `host` meaning wasn't clear to me, and if every package using `numpy` should have this. 

Why would I remove `python >=3.6`?

Reply:
@chrisburr

Reply:
> Why would I remove python >=3.6?

If the package is **not** `noarch: python` it has to be built once per Python version per platform so it is currently built up to 5 times, once for each of `=3.6 cpython*`, `=3.7 cpython*`, `=3.8 cpython*`, `=3.9 cpython*` and `=3.6 pypy*`. Having `>=3.6` breaks this mechanism and `skip: true  # [py<36]` has to be used instead in the build section. If this package **is** `noarch: python` then none of this stuff is done and the version constraint (`>=3.6`) should be applied to `python` directly.

Reply:
Thanks